src/components/chart/specific-chart-options.js

```diff
@@ -42,18 +58,25 @@ class SpecificChartOptions {
     };
 
     adjustChartHeight = (currentChart, percentageHeightDesktop, percentageHeightMobile) => {
-        // get height and width of the plot area
-        const plotHeight = currentChart.plotHeight;
+        // get current width of the plot area
         const plotWidth = currentChart.plotWidth;
-        // calculate the new plot height based on the percentage height
-        // default to the current height
-        let newPlotHeight = plotHeight;
+        let newPlotHeight = undefined;
+        let totalHeight = 400; // Highcharts default height - needed if one of the percentage heights is undefined
```

Comment:
Using the Highcharts default height rather than the current chart height as the default - this means that if there is an undefined value for `percentageHeightmobile`, it won't stay on the desktop height when you resize down (and vice versa),